algorithms/Python/linked_lists/middle-node-linkedlist.py

```diff
@@ -0,0 +1,41 @@
+#printing middle node element of linked list
+#we are using two pointer method to get the solution
+
+class Node:
+    #Constructor to build node
+    def __init__(self, data):
+        self.data = data
+        self.next = None
+
+class LinkedList:
+    #Constructor to create head node
+    def __init__(self):
+        self.head = None
+
+    #Function to push element at head (at begining)
```

Comment:
typo detected by codespell
begining ==> beginning
```suggestion
    #Function to push element at head (at beginning)
```